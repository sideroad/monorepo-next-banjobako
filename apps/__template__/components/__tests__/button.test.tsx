import React from 'react';
import Button from '../button';
import renderer from 'react-test-renderer';

test('Button to render document.head', () => {
  const component = renderer.create(
    <Button onClick={() => {}}>Hello World</Button>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
