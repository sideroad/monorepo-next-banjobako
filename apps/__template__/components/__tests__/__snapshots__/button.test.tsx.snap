// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button to render document.head 1`] = `
<button
  className="jsx-3752195309"
  data-cid="eWRhpRV"
  onClick={[Function]}
>
  Hello World
</button>
`;
